xarray/backends/zarr.py

```diff
@@ -611,6 +611,14 @@ def open_zarr(
     """
     from .api import open_dataset
 
+    if chunks == "auto":
+        try:
+            import dask.array  # noqa
+
+            chunks = {}
+        except ImportError:
+            chunks = None
```

Comment:
Is this a behavior change, or just making the existing behavior more explicit? If I remember correctly, I think it's the later?

Reply:
If this isn't a behavior change, then this is fine to merge.

Reply:
It's to preserve the existing behaviour (to make `open_zarr` backwards compatible).
`open_zarr` uses `open_dataset` and with this PR `open_dataset(..., engine="zarr", chunks="auto")` doesn't work anymore without dask.